routes.py

```diff
@@ -96,12 +115,21 @@ def login():
 
     wallet = hashlib.sha256(keyword.encode()).hexdigest()
 
+    session['wallet'] = wallet
+    session['pubkey'] = pubkey.to_string()
+    session['privkey'] = privkey.to_string()
+
     return jsonify({}), 200
 
 
 @app.route('/getInfo', methods=['GET'])
```

Comment:
`/info` or `/account` looks better. Hypermedia API recommends let URL represents the resource to control. (See also https://spoqa.github.io/2012/02/27/rest-introduction.html)